### DISCUSSION TITLE
Horizontal Bar Charts

### DISCUSSION THREAD

**Discussion by mjanulaitis** [Live-Charts/LiveCharts2#827] [Ideas]
My app currently uses a branched version of LiveCharts to resolve some memory leaks and added support for console apps to generate images (now a feature of v2).  I wasn't able to figure out horizontal bar charts so I just wrote my own since they are really simple given my previous experience writing my own real-time charts in C, C++, C# and even Java.  I want my charts to all have the exact same look and feel so I'd like to figure them out.  Can you please provide a sample if it's possible?  For example in ascii art:

```
Red      | [xxxxxxxx  ]
Blue     | [xxxxxx    ]
Green    | [xxxx      ]
         ---------------
         0              100
```

I'd actually like to be able to move the legend left or right

**Comment by mjanulaitis** [comment]
Alberto any update on this?  I imagine it's really simple and possibly already supported.

**Comment by beto-rodriguez** [comment]
Yes, this is already supported, they are called `RowSeries`, this is a funny sample that uses this series:

https://lvcharts.com/docs/WPF/2.0.0-beta.700/samples.bars.race

![.](https://raw.githubusercontent.com/beto-rodriguez/LiveCharts2/dev/docs/samples/bars/race/result.gif)

Sorry about the late response.
